src/components/ErrorBoundary/BaseErrorBoundary.js

```diff
@@ -2,18 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const propTypes = {
-    /* A message posted to `logError` (along with error data) when this component intercepts an error */
+    /* A message posted to `onError` (along with error data) when this component intercepts an error */
```

Comment:
Renamed because it's not only used for logging anymore.
Naming taken from https://github.com/bvaughn/react-error-boundary